vitrina/redirections/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+
+
+class Redirections(models.Model):
+    objects = models.Manager()
+    created = models.DateTimeField(blank=True, null=True, auto_now_add=True)
+    name = models.TextField(blank=True, null=True)
+    path = models.TextField(blank=True, null=True)
```

Comment:
```suggestion
    path = models.CharField(max_length=255, index=True)
```

This should be a required field. And we want to add index in order to speed up queries.